clang/test/SemaSYCL/accessor_inheritance.cpp

```diff
@@ -1,65 +1,75 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-device -ast-dump %s | FileCheck %s
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -ast-dump -sycl-std=2020 %s | FileCheck %s
+
+// This test checks inheritance support for struct types with accessors
+// passed as kernel arguments, which are decomposed to individual fields.
+
 #include "Inputs/sycl.hpp"
```

Comment:
Hi @Fznamznon , This [PR](https://github.com/intel/llvm/pull/2438) follows `#include "Inputs/sycl.hpp"` style which was suggested to be followed in one of my older PR's. 
Do you want me to change this to system one instead?

Reply:
The PR you mentioned was made before we encountered the problem with missing warnings, and it happened because we don't test front-end with sycl runtime headers included as system headers. After that I suggested to rewrite front-end tests to use system header, to prevent missing warning in the future. See https://github.com/intel/llvm/pull/2575

Reply:
Thanks for the pointer. Will update the tests.